src/main/resources/jupyter/init-actions.sh

```diff
@@ -97,10 +117,16 @@ if [[ "${ROLE}" == 'Master' ]]; then
     JUPYTER_NB_EXTENSIONS=$(jupyterNbExtensions)
     JUPYTER_COMBINED_EXTENSIONS=$(jupyterCombinedExtensions)
     JUPYTER_LAB_EXTENSIONS=$(jupyterLabExtensions)
+    GOOGLE_SIGN_IN_JS_URI=$(googleSignInJsUri)
```

Comment:
Sorry again - these vars look wrong post-rebase.

These should be removed (because they're now in the docker image):
GOOGLE_SIGN_IN_JS_URI
EXTENSION_ENTRY_URI
JUPYTER_LAB_GOOGLE_PLUGIN_URI
SAFE_MODE_JS_URI
EDIT_MODE_JS_URI

This should be added (because it configures the JS code):
JUPYTER_NOTEBOOK_FRONTEND_CONFIG_URI